cmd/livepeer_cli/wizard_eth.go

```diff
@@ -16,3 +19,25 @@ func (w *wizard) setGasPrice() {
 
 	httpPostWithParams(fmt.Sprintf("http://%v:%v/setGasPrice", w.host, w.httpPort), val)
 }
+
+func (w *wizard) signMessage() {
+	fmt.Printf("Enter or paste the message to sign, then press ctrl + d:\n")
+	mystr := []string{""}
+	text, err := w.in.ReadString('\n')
+	mystr = append(mystr, text)
+	for err != io.EOF {
+		if err != io.EOF {
+			text, err = w.in.ReadString('\n')
+			mystr = append(mystr, text)
+		}
+	}
+	message := strings.Join(mystr, "")
+	message = strings.TrimSpace(message)
+	val := url.Values{
+		"message": {fmt.Sprintf("%v", message)},
+	}
+	signedMessage := httpPostWithParams(fmt.Sprintf("http://%v:%v/signMessage", w.host, w.httpPort), val)
+	clipboard.WriteAll(fmt.Sprintf("0x%x", signedMessage))
```

Comment:
Personally not a huge fan of automatic clipboard hijacking (although I do see why it might be convenient). I'd be ok with having the user copy the signed message themselves.